src/automod/Gems.cs

```diff
@@ -59,6 +59,11 @@ private async Task OnReactionAddedAsync(Cacheable<IUserMessage, ulong> message,
         if (gemChannel == null)
             return;
 
+        // Look for a channel called modlog
+        ITextChannel modlogChannel = (await guild.GetTextChannelsAsync())?.FirstOrDefault(x => x.Name == "modlog");
+        if (modlogChannel == null)
```

Comment:
With the addition of this pr, we should create a separate channel called bot_log to not flood the modlog
```suggestion
        // Look for a channel called bot_log
        ITextChannel tc = (await guild.GetTextChannelsAsync())?.FirstOrDefault(x => x.Name == "bot_log");
        if (tc == null)
```